gnosis/safe-react-apps
# Filter native tokens without value

## What it solves
Resolves #221 

## How this PR fixes it
By filtering the native token when `fiatBalance` is zero
If there are no transferable tokens then we are showing a message to the user (See screenshots)

## How to test it
Test in several networks (mainnet, xDai ...)
1) Use a safe with a native token with no value and check that is no listed
2) Use a safe with a native token with value and check that is listed
3) Check the token is included/removed from the final transfer in the safe-web dialog
4) Check the message is showed instead the table if you don't have any transferable asset

## Screenshots

### No assets message

![image](https://user-images.githubusercontent.com/1576776/146518317-22b8a836-92ec-49c2-8ddd-adee3a7dcb81.png)
